types/network/Network.d.ts

```diff
@@ -350,8 +350,15 @@ export class Network {
    * When a String is supplied, the position of the node corresponding to the ID is returned.
    * When nothing is supplied, the positions of all nodes are returned.
    */
-  getPositions(nodeIds?: IdType[]): { [nodeId: string]: Position };
-  getPositions(nodeId: IdType): Position;
+  getPositions(nodeIds?: IdType[] | IdType): { [nodeId: string]: Position };
+  
+  /**
+   * Retrieves the x y position of a specific id.
+   * @param {IdType} id The id to retrieve.
+   * @throws {TypeError} If no id is included.
+   * @throws {ReferenceError} If an invalid id is provided.
```

Comment:
Don't use JSDoc syntax, use TSDoc instead. Param types are already in the code, so:
```
* @param id - The id to retrieve.
```
is enough. Also throws syntax is following (more details in https://github.com/microsoft/tsdoc/pull/175):
```
* @throws {@link TypeError}
* If no id is included.
```
Leave an empty line between the text and the params. And one more thing, your descriptions are quite misleading. For example the param doesn't retrieve the id, as you description suggests, it retrieves the position of the node with given id. Reference error is not thrown if an invalid id is provided but if no node with such id, which may be completely valid, is in the network. The phrase “If no id is included” is also quite weird (included where? how? etc.)